go/kbfs/libkbfs/favorites.go

```diff
@@ -79,12 +81,41 @@ type favReq struct {
 	ctx context.Context
 }
 
+type favoriteData struct {
+	Name         string
+	FolderType   keybase1.FolderType
+	ID           keybase1.TLFID
+	Private      bool
+	TeamID       keybase1.TeamID
+	ResetMembers []keybase1.User
+}
+
+func favoriteDataFrom(folder keybase1.Folder) favoriteData {
+	return favoriteData{
+		Name:       folder.Name,
+		FolderType: folder.FolderType,
+		// TODO: get ID??
+		Private: folder.Private,
+		TeamID:  *folder.TeamID,
+	}
+}
+
+type homeTLFInfo struct {
+	PublicID      keybase1.TLFID
+	PrivateID     keybase1.TLFID
+	PublicTeamID  keybase1.TeamID
+	PrivateTeamID keybase1.TeamID
+}
+
 // Favorites manages a user's favorite list.
 type Favorites struct {
 	config   Config
 	disabled bool
 	log      logger.Logger
 
+	// homeTLFInfo stores the IDs for the logged-in user's home TLFs
+	homeTLFInfo homeTLFInfo
```

Comment:
Looks like this doesn't get set yet?  I'm guessing you'll need another request type or channel for that, or just use a lock maybe.